Packs/CortexXDR/Playbooks/Cortex_XDR_Alerts_Handling_v2_README.md

```diff
@@ -6,23 +6,26 @@ Supported alert categories:
 - Cloud Token Theft
 - RDP Brute-Force
 - First SSO Access
-- Cloud IAM User Access Investigation.
+- Cloud IAM User Access Investigation
+- Identity Analytics
 
 ## Dependencies
 
 This playbook uses the following sub-playbooks, integrations, and scripts.
 
 ### Sub-playbooks
 
-* Cortex XDR - Malware Investigation
-* Cortex XDR - XCloud Token Theft Response
-* Cortex XDR - Possible External RDP Brute-Force
 * GenericPolling
+* Cortex XDR - First SSO Access
 * Cortex XDR - XCloud Cryptojacking
-* Cortex XDR - Cloud Data Exfiltration Response
-* Cortex XDR - Cloud IAM User Access Investigation
+* Cortex XDR - XCloud Token Theft Response
 * Cortex XDR - Port Scan - Adjusted
-* Cortex XDR - First SSO Access
+* Cortex XDR - Cloud IAM User Access Investigation
+* Cortex XDR - Malware Investigation
+* Cortex XDR - Possible External RDP Brute-Force
+* Cortex XDR - Cloud Data Exfiltration Response
+* Cortex XDR Remote PsExec with LOLBIN command execution alert
```

Comment:
is this missing a hyphen?
Cortex XDR - Remote PsExec with LOLBIN command execution alert

Reply:
This is the playbook name